src/Bridges/Constraint/vectorize.jl

```diff
@@ -61,62 +150,112 @@ end
 function MOI.supports(
     ::MOI.ModelLike,
     ::Union{MOI.ConstraintPrimalStart, MOI.ConstraintDualStart},
-    ::Type{<:VectorizeBridge})
-
+    ::Type{<:VectorizeBridge},
```

Comment:
No need for last comma

Reply:
This is JuliaFormatter style. I'm keeping. Will be resolved once we start using JuliaFormatter.